frontend/src/components/EditAssembly.tsx

```diff
@@ -49,10 +74,23 @@ export function EditAssembly(state: { group: UserDataGroupType }) {
     fetch().catch(console.error);
   }, [statusChanges]);
 
+  // Triggered when a votation is created, edited, deleted (started/stopped).
   useEffect(() => {
     const fetch = async () => {
       const votes = await getVotations(group.groupSlug);
       setVotations(votes);
+
+      // Set number of submitted votes on current votation when page is initially loaded.
+      // Using websocket to update this value in real time.
+      let currentVotes = 0;
+      votes.forEach((vote) => {
+        if (vote.isActive) {
+          vote.options.forEach((option) => {
+            currentVotes += option.voteCount;
+          });
+        }
+      });
+      setSubmittedVotes(currentVotes);
     };
 
     fetch().catch(console.error);
```

Comment:
Should maybe be handled?

Reply:
Yep, throw to sentry in the future? (#150)?